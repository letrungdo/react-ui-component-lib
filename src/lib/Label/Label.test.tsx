import { mount } from "enzyme";
import React from "react";
import { Label } from ".";

describe("Label", () => {
    it("No text", () => {
        const elm = mount(<Label />);

        expect(elm).toMatchSnapshot();
    });

    it("Has text", () => {
        const elm = mount(<Label>Hello</Label>);

        expect(elm).toMatchSnapshot();
    });
});
