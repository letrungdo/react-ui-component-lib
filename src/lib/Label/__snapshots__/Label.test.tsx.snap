// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label Has text 1`] = `
.emotion-0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  white-space: nowrap;
  width: auto;
  font-family: RobotoRegular;
}

.emotion-0.inactive {
  opacity: 0.5;
}

.emotion-0.text {
  white-space: pre-wrap;
  word-break: break-word;
}

<Styled(LabelComponent)
  defaultValue="-"
  fontFamily="RobotoRegular"
>
  <LabelComponent
    className=" emotion-0"
    defaultValue="-"
    fontFamily="RobotoRegular"
  >
    <p
      className="label  emotion-0"
    >
      Hello
    </p>
  </LabelComponent>
</Styled(LabelComponent)>
`;

exports[`Label No text 1`] = `
.emotion-0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  white-space: nowrap;
  width: auto;
  font-family: RobotoRegular;
}

.emotion-0.inactive {
  opacity: 0.5;
}

.emotion-0.text {
  white-space: pre-wrap;
  word-break: break-word;
}

<Styled(LabelComponent)
  defaultValue="-"
  fontFamily="RobotoRegular"
>
  <LabelComponent
    className=" emotion-0"
    defaultValue="-"
    fontFamily="RobotoRegular"
  >
    <p
      className="label  emotion-0"
    >
      -
    </p>
  </LabelComponent>
</Styled(LabelComponent)>
`;
